# world origin values (altitude is AMSL rather than WGS84 of GPS)

world_origin_latitude_deg: 39.980104  # degree
world_origin_longitude_deg: 116.345922  # degree
world_origin_AMSL_alt_metre: 0.0  # metre


# PX4 parameters

MPC_THR_HOVER: 0.5
MPC_THR_MIN: 0.12
MPC_THR_MAX: 1.0

MPC_Z_P: 1.0
MPC_Z_VEL_P_ACC: 4.0
MPC_Z_VEL_I_ACC: 2.0
